assets/app/styles/_components.less

```diff
@@ -4,7 +4,6 @@
 
 .components {
   border-left: 3px solid #ccc;
-  border-left-style: dotted;
```

Comment:
This changes the style when a replication controller is not under a service. Was that intentional? It used to have a dotted line.

![openshift_web_console](https://cloud.githubusercontent.com/assets/1167259/13986316/304c7b2a-f0d8-11e5-98f9-00a26fa734a7.png)

Now it doesn't:

![openshift_web_console](https://cloud.githubusercontent.com/assets/1167259/13986324/39517b08-f0d8-11e5-90ae-c7f90c2e4965.png)